application-licensing-licensor/application-licensing-licensor-api/src/test/java/com/xwiki/licensing/internal/DefaultLicensingConfigurationTest.java

```diff
@@ -75,4 +96,22 @@ public void getBlocklistWithEmptyList() throws Exception
 
         assertEquals(Collections.emptyList(), mocker.getComponentUnderTest().getAutoUpgradeBlocklist());
     }
+
+    @Test
+    public void getLocalStorePath() throws Exception
+    {
+        when(this.configurationSource.getProperty("licensing.localStorePath")).thenReturn("path");
+        this.mocker.getComponentUnderTest().getLocalStorePath();
+
+        verify(this.environment, never()).getPermanentDirectory();
```

Comment:
``verify`` is normally used for **void** methods which is not the case here. I'm not sure it's worth verifying if ``getPermanentDirectory`` is called. I would simply assert the value returned by ``getLocalStorePath``.